printing/ppextend.mli

```diff
@@ -35,7 +35,7 @@ type unparsing =
   | UnpMetaVar of entry_relative_level * Extend.side option
   | UnpBinderMetaVar of entry_relative_level * pattern_quote_style
   | UnpListMetaVar of entry_relative_level * unparsing list * Extend.side option
-  | UnpBinderListMetaVar of bool * unparsing list
+  | UnpBinderListMetaVar of bool * bool * unparsing list
```

Comment:
Perhaps a small comment saying that the booleans are respectively for isopen and noquote.
BTW, wouldn't quote be easier to use than the negative noquote? (well, ideally, those boolean should probably be adhoc variant types with two constructors)

(P.S. : the comment should probably rather be in the mli)

Reply:
I will add a comment and negate the quote boolean. 

I will also adopt your other suggestions.